config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```
@@ -3359,6 +3359,51 @@ presubmits:
             cpu: 4
             memory: 6Gi
 
+  - name: pull-kubernetes-node-e2e-resources-health
```

```suggestion
  - name: pull-kubernetes-node-e2e-resource-health-status
```

I would ideally mark it as close as possible to the feature gate.